src/main/resources/com/hp/application/automation/tools/run/RunFromFileBuilder/config.jelly

```diff
@@ -38,6 +38,17 @@
     <f:entry title="Timeout" field="fsTimeout">
         <f:textbox name="runfromfs.fsTimeout" value="${instance.runFromFileModel.fsTimeout}"/>
     </f:entry>
+
+    <f:entry title="Run Mode" field="fsUftRunMode">
+        <select name="runPipeline.fsUftRunMode">
```

Comment:
RunFromFileBuilder Jelly also appears in regular jobs
Its joined for LR and UFT.
That's why LR options are also here...
That's why i didn't want this to be a pipeline step- and have LR and UFT separated in pipeline